documentation/specs/src/interoperability/ethereum-bridge/transfers_to_ethereum.md

```diff
@@ -0,0 +1,135 @@
+and posted on Namada.
+
+It is incumbent on the end user to  request an appropriate ["proof"](proofs.md) 
+of the transaction. This proof must be submitted to the appropriate Ethereum smart
+contract by the user to redeem Ethereum assets / mint wrapped assets. This also
+means all Ethereum gas costs are the responsibility of the end user.
+
+A relayer binary will be developed to aid users in accessing the proofs
+generated by Namada validators as well as posting this proof to Ethereum. It
+will also aid in batching transactions.
+
+## Moving value to Ethereum
+
+To redeem wrapped Ethereum assets, a user should make a transaction to burn
+their wrapped tokens, which the `#EthBridge` validity predicate will accept.
+
+Mints of a wrapped Namada token on Ethereum (including NAM, Namada's native token)
+will be represented by a data type like:
+```rust
+struct MintWrappedNam {
+    /// The Namada address owning the token
+    owner: NamadaAddress,
+    /// The address on Ethereum receiving the wrapped tokens
+    receiver: EthereumAddress,
+    /// The address of the token to be wrapped 
+    token: NamadaAddress,
+    /// The number of wrapped Namada tokens to mint on Ethereum
+    amount: Amount,
+}
+```
+
+If a user wishes to mint a wrapped Namada token on Ethereum, they must
+submit a transaction on Namada that:
+- stores `MintWrappedNam` on chain somewhere - TBD
+- sends the correct amount of Namada token to `#EthBridgeEscrow`
+
+## Batching
+
+Ethereum gas fees make it prohibitively expensive in many cases to submit
+the proof for a single transaction over the bridge. Instead, it is typically
+more economical to submit proofs of many transactions in bulk. This batching
+is described in this section.
+
+A pool of transaction from Namada to Ethereum will be kept by Namada. Every
+transaction to Ethereum that Namada validators approve will be added to this
+pool. We call this the _Bridge Pool_.
+
+The Bridge Pool should be thought of as a sort of mempool. When users who
+wish to move assets to Ethereum submit their transactions, they will pay some
+additional amount of NAM (of their choosing) as a way of covering the gas
+costs on Ethereum. Namada validators will hold these fees in a Bridge Pool
+Escrow.
+
+When a batch of transactions from the Bridge Pool is submitted by a user to
+Ethereum, Namada validators will receive notifications via their full nodes.
+They will then pay out the fees for each submitted transaction to the user who
+relayed these transactions (still in NAM). These will be paid out from the
+Bridge Pool Escrow.
+
+The idea is that users will only relay transactions from the Bridge Pool
+that make economic sense. This prevents DoS attacks by underpaying fees as
+well as obviating the need for Ethereum gas price oracles. It also means
+that transfers to Ethereum are not ordered, preventing other attack vectors.
+
+The Bridge Pool will be organized as a Merkle tree. Every time it is updated,
```

Comment:
to clarify, it's a Merkle tree of height 1 including all `TransferToEthereum`s that have ever been added to `/pending_transfers` (in the order in which they were added)?

Reply:
I haven't decided on the underlying tree type yet. I'm actually leaning toward binary so I can reuse existing third party code. It will also likely store the transfers in order of their hashes (so a random order). We remove transfers once we are notified by ethereum that they have been relayed or they time out.

Reply:
Anyway, this is an implementation detail that doesn't affect the final functionality, so it doesn't need to be included in the specs.